### ISSUE TITLE
Webs!

### ISSUE THREAD

**Issue by Lagicrus** [Bouncyrock/TaleSpire-Beta-AssetRequests#128]
**Describe the Asset**
What does it look like, how is it used. Paint us a figurative picture.
Webs for spiders to crawl over, from simple 1x1 flat ones to ones that fill up space and are as tall as they are wide.
So you can have both webs for the players/spiders to walk over. As well as ones in Minecraft, for example, that fill up the whole area when there are that many webs.

**Examples (images, videos, something which gets the point across**
Or show us a literal picture. That works too.
![image](https://user-images.githubusercontent.com/24227350/93669321-57e14280-fa8b-11ea-8483-9087fe6e7139.png)
![image](https://user-images.githubusercontent.com/24227350/93669355-8fe88580-fa8b-11ea-81bd-eea8d9a9080f.png)
![image](https://user-images.githubusercontent.com/24227350/93669358-94ad3980-fa8b-11ea-8c0d-e5e1f7420488.png)

** ADD :thumbsup:[thumbs up reaction] to give this request more attention.**

**Comment by svenboulanger** [comment]
Also, in the same theme: a webbed cocoon (e.g. for spider nests or just unfortunate adventurers).

**Comment by otting** [comment]
yes we need this, we already have spiders, but they are really sad without their webs

**Comment by madcrammer** [comment]
Would love to see webs - lots of campaign maps use them and as otting mentioned we already have spiders....
